syft/distro/identify.go

```diff
@@ -84,7 +85,8 @@ func assemble(name, version string) *Distro {
 	distroType, ok := IDMapping[name]
 
 	// Both distro and version must be present
-	if len(name) == 0 || len(version) == 0 {
+	if len(name) == 0 {
+		log.Debugf("Name (%s) not found", name)
```

Comment:
nit: logging isn't necessary here

Reply:
I used it for some local debugging but wasn't sure if it'd be useful later on for troubleshooting! Will remove.